src/components/MediaHandler/MediaHandler.js

```diff
@@ -51,11 +54,10 @@ export class MediaHandler extends React.Component {
             flexDirection: "column",
           }}
         >
-          <button onClick={this.obtainMedia}>Add Media</button>
+          <button onClick={this.obtainMedia}>{buttonText}</button>
         </div>
       );
     } else {
-      const { mime, error } = this.state;
       const fileLocation = `file://${mediaLocation}`;
 
       try {
```

Comment:
In retrospect, I think copying and pasting the div and button was a bad implementation.

can you reimplement it so that it renders the content dynamically?

for example, right before the try statement
```
const content = (() => {
if (_.isNil(mime)) {
return null
} else if (error) {
return <h4>Failed to read file</h4>
} else if (mime includes "image) {
return <img .... />
}
.
.
.
})()
```

then you can just render it as 
```
<div>
  {content}
  <button>
</div>
```

reminder to do this in a separate commit from the button text one